explainers/beforematch.md

```diff
@@ -404,6 +404,58 @@ could be applied to the entire ancestor chain.
   impossible to provide the right selector which can modify a style on an
   unrelated element.
 
+#### `<details>`/`<summary>` auto-expanding
+Instead of having a generic hidden-matchable property or a specialized
+beforematch event, we could just slightly tweak the `<details>` element to make
+collapsed content searchable. This addresses most of the use cases we have seen
+for the beforematch event, and the details element already has a toggle event
+which would act like the beforematch event. This could be implemented either by
+adding an attribute to the details element saying that the collapsed content
+should be searchable, or just by making it searchable as-is because find-in-page
+is not specced.
```

Comment:
If you want you can add a code example here to show what the change would be something like

```html
<details searchable>
  <summary>Summary content</summary>
  Detailed content
</details>
```

Reply:
Thanks! done